autoemulate/experimental/compare.py

```diff
@@ -21,7 +21,9 @@ def __init__(
         y: InputLike,
         models: list[type[Emulator]] | None = None,
         device: DeviceLike | None = None,
+        random_seed: int | None = None,
     ):
+        self.random_seed = random_seed
```

Comment:
I think we might need to set the random seed within this init since there is a random_split call on L42?